nixos/tests/web-apps/linkding/defaults.nix

```diff
@@ -0,0 +1,19 @@
+import ../../make-test-python.nix (
+  { lib, ... }:
+  let
+    nodeName = "server";
+  in
+  {
+    name = "linkding-defaults";
+
+    nodes."${nodeName}" = {
+      services.linkding.enable = true;
+    };
+
+    testScript = ''
+      ${nodeName}.wait_for_unit("linkding.service")
```

Comment:
Would also be good to request "/" or some other path, just to make sure it responds to requests too